scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -447,7 +448,7 @@
               total-pods (-> @all-pods-atom keys count)
               total-nodes (-> @current-nodes-atom keys count)
               {:keys [image user command max-pods-outstanding max-total-pods max-total-nodes]
-               :or {command "exit 0" max-total-pods 32000 max-total-nodes 1000}} synthetic-pods-config]
+               :or {command "sleep 300" max-total-pods 32000 max-total-nodes 1000}} synthetic-pods-config]
```

Comment:
The default for `clobber-synthetic-pods` appears to be `nil`, so it seems like the default for `command` should continue to be `exit 0`, so that the two defaults make sense together.

Reply:
Good catch!